src/content/docs/zh-cn/reference/configuration-reference.mdx

```diff
@@ -646,6 +646,14 @@ export default defineConfig({
 }
 ```
 
+### image.domains
+
+占位
```

Comment:
When should we update the placeholders here

Reply:
@liruifengv I think everything to do with configuration-reference, since there will be other changes there too, can go in the other big PR!